stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -137,29 +143,76 @@ There is no default size to preserve what's being ultimately rendered.
 
 ## Colors
 
-The different colors available are: `text` (default), `highlighted-text`, `info`, `success`, `warning`, `error` which
-all refer to OKP4 palette and theming.
+There are different colors available referring to OKP4 palette and themes.
+
+The base colors that can handle the most use cases:
+
+- `text` - the default text color,
+- `highlighted-text`,
+- `info`,
+- `success`,
+- `warning`,
+- `error`
+
+To allow the management of higher contrasts or cases where the color must remain the same according to the theme, the OKP4 palette addresses these issues by adding 2 more complex color effects:
```

Comment:
I would have moved the text about the complex colors part after the screenshot of the basic colors, like this:
- text about basic colors
- screenshot 
- text about the complex colors
- screenshot
I think it's more readable, but that's just my opinion ;)

Reply:
To do that, we have to separate the 2 stories in different `Canvas`. What do you think @fredericvilcot  ?

Reply:
Yes , I agree with @lolottetheclash 's remark, we could make 2 `Story` in 2 dedicated `Canvas` :)

Reply:
ok great let's do that !